topics/tutorials/code_inspections.md

```diff
@@ -126,59 +101,42 @@ Implicit in using [`LocalInspectionTool`](%gh-ic%/platform/analysis-api/src/com/
 >
 > `See <em>Includes</em> tab in <a href="settings://fileTemplates">Settings | Editor | File and Code Templates</a> to configure.`
 >
 
-### Inspection Unit Test
+### Inspection Test
 
 > Please note that running the test requires setting system property `idea.home.path` in the `test` task configuration of the Gradle build script.
 >
 {style="note"}
 
-The `comparing_references_inspection` code sample provides a unit test for the inspection.
+The `comparing_string_references_inspection` code sample provides a test for the inspection.
 See the [](testing_plugins.md) section for general information about plugin testing.
 
-The `comparing_references_inspection` test is based on the [`UsefulTestCase`](%gh-ic%/platform/testFramework/src/com/intellij/testFramework/UsefulTestCase.java) class, part of the JUnit framework APIs.
+The `comparing_string_references_inspection` test is based on the [`UsefulTestCase`](%gh-ic%/platform/testFramework/src/com/intellij/testFramework/UsefulTestCase.java) class, part of the JUnit framework APIs.
 This class handles much of the underlying boilerplate for tests.
 
 By convention, the folder <path>test/testData/</path> contains the test files.
-The folder contains pairs of files for each test using the name convention <path>∗.java</path> and <path>∗.after.java</path>.
+The folder contains pairs of files for each test using the name convention <path>∗.java</path> and <path>∗.after.java</path>, e.g., <path>Eq.java</path> / <path>Eq.after.java</path>.
 
-In the case of `comparing_references_inspection` the test files are <path>Eq.java</path> / <path>Eq.after.java</path>, and <path>Neq.java</path> / <path>Neq.after.java</path>.
+The `comparing_string_references_inspection` tests run the inspection on the <path>∗.java</path> files, apply the quick fix, and compare the results with the respective <path>∗.after.java</path> files containing expected results.
 
-The `comparing_references_inspection` tests run the inspection on the <path>∗.java</path> files, implement the quick fix, and compare the results with the respective <path>∗.after.java</path> files containing expected result.
+## Running the Comparing String References Inspection Code Sample
 
-## Running the Comparing References Inspection Code Sample
-
-The [comparing_references_inspection](%gh-sdk-samples%/comparing_references_inspection) code sample adds a new inspection to the <control>Java | Probable Bugs</control> group in the [Inspections List](https://www.jetbrains.com/help/idea/inspections-settings.html).
+The [comparing_string_references_inspection](%gh-sdk-samples%/comparing_string_references_inspection) code sample adds a new inspection to the <control>Java | Probable Bugs</control> group in the [Inspections](https://www.jetbrains.com/help/idea/inspections-settings.html) configuration.
 
 See [](code_samples.md) on how to set up and run the plugin.
 
-### Configuring the Plugin
```

Comment:
why removed?

Reply:
I think that it doesn't make sense after removing custom options panel. All that can be done now is configuring severity, which is covered under https://www.jetbrains.com/idea/webhelp/code-inspection.html linked in `tldr`. IMHO, it would be just a noise in this doc.